app/src/examples/WorkletFactoryCrashExample.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React from 'react';
+import { Button, StyleSheet, View, Text } from 'react-native';
+
+export default function App() {
+  function handleOnPress() {
+    function badWorklet() {
+      'worklet';
+      // @ts-expect-error
+      unexistingVariable;
+    }
+  }
+
+  return (
+    <View style={styles.container}>
+      <Text style={styles.text}>
+        Clicking the button below should give a stack trace that denotes the
+        error in `badWorklet` function, pointing to `unexistingVariable` usage.
```

Comment:
Instead of providing the instructions here, could we `try/catch` the exception and check the stack trace (as a string) automatically? If this works, let's just `console.log('OK')`.

Reply:
@tomekzaw Unfortunately this cannot be done like this. The stack trace is an unrefined error that points to the bundled code instead of the source code. We need the redbox to do its magic and point to the correct place.